com.endlessm.HatchPreviewer.json

```diff
@@ -1,10 +1,10 @@
 {
     "app-id": "com.endlessm.HatchPreviewer",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "1.6",
-    "base": "io.atom.electron.BaseApp",
-    "base-version": "stable",
-    "branch": "stable",
+    "runtime-version": "18.08",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "18.08",
+    "branch": "wip",
```

Comment:
I understand what you're doing here, but this won't get through the flathub build system ;-) switch it back and then I can review and merge after the upstream PR

Reply:
I was testing you! (not). Good catch, thanks.